common/vulkan_wrapper.cpp

```diff
@@ -1048,6 +1085,38 @@ PFN_vkCreateScreenSurfaceQNX vkCreateScreenSurfaceQNX;
 PFN_vkGetPhysicalDeviceScreenPresentationSupportQNX vkGetPhysicalDeviceScreenPresentationSupportQNX;
 #endif
 
+#ifdef VK_USE_PLATFORM_SCREEN_QNX
+PFN_vkGetPhysicalDeviceVideoCapabilitiesKHR vkGetPhysicalDeviceVideoCapabilitiesKHR;
+PFN_vkGetPhysicalDeviceVideoFormatPropertiesKHR vkGetPhysicalDeviceVideoFormatPropertiesKHR;
+PFN_vkCreateVideoSessionKHR vkCreateVideoSessionKHR;
+PFN_vkDestroyVideoSessionKHR vkDestroyVideoSessionKHR;
+PFN_vkGetVideoSessionMemoryRequirementsKHR vkGetVideoSessionMemoryRequirementsKHR;
+PFN_vkBindVideoSessionMemoryKHR vkBindVideoSessionMemoryKHR;
+PFN_vkCreateVideoSessionParametersKHR vkCreateVideoSessionParametersKHR;
+PFN_vkUpdateVideoSessionParametersKHR vkUpdateVideoSessionParametersKHR;
+PFN_vkDestroyVideoSessionParametersKHR vkDestroyVideoSessionParametersKHR;
+PFN_vkCmdBeginVideoCodingKHR vkCmdBeginVideoCodingKHR;
+PFN_vkCmdEndVideoCodingKHR vkCmdEndVideoCodingKHR;
+PFN_vkCmdControlVideoCodingKHR vkCmdControlVideoCodingKHR;
+#endif
+
+#ifdef VK_USE_PLATFORM_SCREEN_QNX
+PFN_vkCmdDecodeVideoKHR vkCmdDecodeVideoKHR;
+#endif
+
+#ifdef VK_USE_PLATFORM_SCREEN_QNX
+#endif
```

Comment:
What's with the empty `ifdef`?

Reply:
I'm not too concerned about the empty `#ifdef`s, but looking closer, the video content should be wrapped with `VK_ENABLE_BETA_EXTENSIONS` instead of `VK_USE_PLATFORM_SCREEN_QNX`. I think generate_vulkan_wrapper.py isn't detecting the correct platform guard for beta content.

Reply:
Pushed a fix in debeff612dfaaa338e0d705423501a5ad63c05e5 to for the beta platform guards and it was easy enough strip out empty `#ifdef` blocks at the same time.